config/routes.rb

```diff
@@ -1,14 +1,18 @@
 Rails.application.routes.draw do
+  get 'posts/new'
```

Comment:
- [ ] There are two ways for defining routes [Resourceful routes](https://guides.rubyonrails.org/routing.html#crud-verbs-and-actions) or [Non-resourceful routes](https://guides.rubyonrails.org/routing.html#non-resourceful-routes). Let's choose only one of them for posts routes.

Reply:
Always have asked what the resources do. Thank you. Corrected